stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 11.10.0
+version: 11.10.1
```

Comment:
And because you're adding a feature - could you also please set the version to: 11.11.0.